boringtun/src/noise/timers.rs

```diff
@@ -310,13 +310,8 @@ impl TunnInner {
         if self.sessions[current_session % super::N_SESSIONS].is_some() {
             let time_current = Instant::now().duration_since(self.timers.time_started);
```

Comment:
The time subtraction going on here is NOT correct. Let's understand this before trying to fix it.